spec/features/publish/courses/editing_course_outcome_spec.rb

```diff
@@ -50,8 +50,21 @@
         and_there_is_a_tda_course_i_want_to_edit
         when_i_visit_the_course_outcome_page_in_the_next_cycle
         and_i_choose_qts
+        and_the_back_link_points_to_the_outcome_page
         and_i_choose_the_fee_paying
+        and_the_back_link_points_to_the_funding_type_page
         and_i_choose_part_time
+        and_the_back_link_points_to_the_study_mode_page
+
+        and_i_click_back
+        then_i_am_on_the_study_mode_page
+
+        and_i_click_back
+        then_i_am_on_the_funding_type_page
+
+        and_i_update
+        and_i_update
```

Comment:
```suggestion
        and_i_update
```
```suggestion
        when_i_update
        and_i_update
```

This might make it clear that you didn't just accidentally copy and paste twice.

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/4284/commits/7a007a75f39a628f5e01434428272e7d3e80323d